source/administration/config-settings.rst

```diff
@@ -391,14 +370,44 @@ Restrict the permission level required to delete private groups. Deleted groups
 | This feature's ``config.json`` setting is ``"RestrictPrivateChannelDeletion": "all"`` with options ``all``, ``team_admin`` and ``system_admin`` for above settings respectively.   |
 +------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Allow which users to delete messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Restrict the permission level required to delete messages.
+
+**Message authors can delete their own messages, and Administrators can delete any message**: Allow authors to delete their own messages, and allow Administrators to delete any message.
+
+**Team Admins and System Admins**: Allow only Team Admins and System Admins to delete messages.
+
+**System Admins**: Allow only System Admins to delete messages.
+
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"RestrictPostDelete": "all"`` with options ``all``, ``team_admin`` and ``system_admin`` for above settings respectively.               |
++------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Allow users to edit their messages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Set the time limit that users have to edit their messages after posting.
+
+**Any time**: Allow users to edit their messages at any time after posting.
+
+**Never**: Do not allow users to edit their messages.
+
+**nnn seconds after posting**: Users can edit their messages within the specified time limit after posting.
```

Comment:
Maybe *{n} seconds after posting*. Using n for number and x for letters is a common convention in tech docs.